src/datastore/Restaurant.cpp

```diff
@@ -0,0 +1,270 @@
+/**
+ * CS1D - Fast Food Associates
+ *
+ * Implemention of the Restaurant class that provides a means
+ * for validating restaurants, string menu items and distances
+ * to other restaurants
+ *
+ * @author   edt
+ */
+
+
+#include "Restaurant.h"
+#include <string>
+#include <ostream>
+#include <iostream>
+
+int Restaurant::sm_NextNumber = 1;

```

Comment:
Can we get a comment saying what this is?

Example:
```suggestion
//Static member of Restaurant
int Restaurant::sm_NextNumber = 1;
```